generators/webapp/templates/server/plugins/webapp/index.js

```diff
@@ -63,26 +89,36 @@ function makeRouteHandler(options, userContent) {
       return WEBPACK_DEV ? devJSBundle : assets.js && `/js/${assets.js}` || "";
     };
 
+    const bundleManifest = () => {
+      return assets.manifest ? `/js/${assets.manifest}` : "";
+    };
+
     const callUserContent = (content) => {
       const x = content(request);
       return !x.catch ? x : x.catch((err) => {
-        return Promise.reject({
```

Comment:
we want to render the page with the error message?  Then we might want to make it consistent to do the same in the catch below.  You can add this as an issue for later instead of this PR though.